src/main/kotlin/ractingCar/entity/Car.kt

```diff
@@ -0,0 +1,17 @@
+package ractingCar.entity
+
+class Car(var moves: Int = 0) {
```

Comment:
```suggestion
class Car(val moves: Int = 0) {
```
외부에서 move를 변경할 수 있네요.
외부에서 값을 변경하게 되면 캡슐화가 깨지게 되며, 외부에서 변경함에 따라 의도치 않게 동작할 수 있어요.
외부에서 변경이 불가능하도록 해보면 어떨까요?

Reply:
```
class Car(moves: Int = INITIAL_MOVES) {
    private var privateMoves = moves
    val moves get() = privateMoves

    fun move(rand: Int) {
        if (rand >= MINIMUM_MOVE_FORWARD_NUMBER) this.privateMoves += 1
    }
}
```
이렇게 수정했습니다

Reply:
이런 식으로 parameter를 받아 내부 값으로 전달하고 private setter를 사용할 수 없게 설정했습니다.
```
class Car(moves: Int = INITIAL_MOVES) {
    var moves: Int = moves
        private set

    // 값을 넣어서 움직이는 함수로 수정
    fun move(rand: Int) {
        if (rand >= MINIMUM_MOVE_FORWARD_NUMBER) this.moves += 1
    }
}
```